File: 01_Fizz Buzz/Ideal Solutions/fizzBuzz.cpp
#include<bits/stdc++.h>
using namespace std;
void fizzBuzz(int n) {
	// Function to implement Fizz Buzz
	for (int i = 1; i <= n; i++) {
		if (i % 3 == 0 && i % 5 != 0)
			cout << "Fizz" << " ";
		if (i % 5 == 0 && i % 3 != 0)
			cout << "Buzz" << " ";
		if (i % 5 == 0 && i % 3 == 0)
			cout << "FizzBuzz" << " ";
		if (i % 5 != 0 && i % 3 != 0)
			cout << i << " ";
	}
}
int main() {
	int n = 10;
	fizzBuzz(n);
	return 0;
}

File: 01_Fizz Buzz/Ideal Solutions/fizzBuzz.py
def fizzBuzz(N):
    for value in range(1, N + 1):

        # For multiple of 5 and 3 ----> Print "FizzBuzz"
        if value % 5 == 0 and value % 3 == 0:
            print("FizzBuzz")
            continue

        # For multiple of 5 ----> Print "Buzz"
        if value % 5 == 0:
            print("Buzz")
            continue

        # For multiple of 3 ----> Print "Fizz"
        if value % 3 == 0:
            print("Fizz")
            continue
        print(value)
 




if __name__ == "__main__":

    # Static Input
    N = 7

    fizzBuzz(N)
